### DISCUSSION TITLE
Nuxt replace startup/restart page

### DISCUSSION THREAD

**Discussion by waradu** [nuxt/nuxt#30027] [Questions]
I want to replace this screen with a custom HTML file:

![nuxt startup page](https://github.com/user-attachments/assets/646c2561-e72a-4b4b-9a90-d9f277b2ead1)

Why? I use Nuxt with Tauri and want some part of the loading screen to have "data-tauri-drag-region" so I can drag it even in the loading state.

I have already tried to change the "spaLoadingTemplate" or "devServer" > "loadingTemplate" but this didn't work.

**Comment by dolbex** [comment]
I would also be interested to know how this is possible through a module
